src/milia/api/io.clj

```diff
@@ -39,12 +39,15 @@
 (defn- req+auth
   "Add authorization to options"
   [req]
-  (let [{:keys [temp-token token username password]} *credentials*]
+  (let [{:keys [temp-token token username password]} *credentials*
+        {:keys [auth-token]} req]
```

Comment:
pre-existing but we should call `req` `http-options` here and in the `build-request` function, that's what it really is